src/main/java/nl/mpcjanssen/simpletask/Simpletask.kt

```diff
@@ -459,6 +460,11 @@ class Simpletask : ThemedActivity() {
                 setTitle(R.string.title_filter_drawer)
             }
             Mode.SELECTION -> {
+                val actionColor = ContextCompat.getDrawable(this, R.color.gray81)
+                actionBar.setBackgroundDrawable(actionColor)
+
+                /* window.setStatusBarColor(R.color.gray87) */
```

Comment:
This was not working and was messing up the statusbar color (it always set it to black), so commented out for the time being.